### PR TITLE
Shuffle code and testing complete

### PR DESCRIPTION
**Pull request by adnanhemani**
[X] Wrote test for feature
[X] Added changes in the Changelog section in README.md
[X] Bumped version number (delete if unneeded)

**Changes proposed:**
Changes requested for PR #340.

### CODE REVIEW

**Comment by adnanhemani** [comment]
Travis continues to fail due to current state of master.

**Comment by adnanhemani** [comment]
@rameshvs Could you please review this PR when you have time? Thanks!

File: tests/test_tables.py [hunk 1]
```diff
@@ -1196,6 +1196,38 @@ def test_sample_weights_with_none_k(table):
...
+    """
+    Tests that all rows do not always appear in the same order
+    as they were in the original table
+    """
+    # This is more or less a regression test - you could get very 
+    # unlucky and have no change in the order of the rows by complete 
+    # accident. However, this is highly unlikely to happen.
+    
+    original_order = table.column("letter")
+    print(original_order)
```
Comment (davidwagner) [inline tests/test_tables.py]: Super-minor: Maybe we shouldn't `print()` anything in the tests, to avoid cluttering the output of `make test`?

Comment (adnanhemani) [reply tests/test_tables.py]: Agreed. Sorry about that! Pushing change now!

**Pull request merged**